### ISSUE TITLE
Conv2d_transpose requires asymmetric padding which the CUDA EP currently does not support

### ISSUE THREAD

**Issue by ultimatedigiman** [microsoft/onnxruntime#11312]
**Describe the bug**
After successfully converting a tensorflow frozen graph(.pb) into a .oonx model, I encountered a warning while inferring the onnx model with onnxruntime-gpu in python.
The warning says **conv2d_transpose** op is not supported because it requires **asymmetric padding** which the CUDA EP currently does not support.

Bur it's strange that I found  **ConvTranspose** is in [Supported ONNX Operators](https://github.com/onnx/onnx-tensorrt/blob/main/docs/operators.md) and it seems [the asymmetric padding issue](https://github.com/microsoft/onnxruntime/pull/4627) is already solved.

**System information**
- OS Platform and Distribution:Linux Ubuntu 20.04
- ONNX Runtime installed from: `pip install onnxruntime-gpu`
- ONNX Runtime version: 1.10.0
- Python version: 3.6
- CUDA/cuDNN version: 11.3/8.2.1
- Tensorflow Version: 1.15
- opset: 13

**Expected behavior**
**conv2d_transpose** has been supported.

**Screenshots**
![image](https://user-images.githubusercontent.com/67790229/164643616-00ca3619-ded4-4390-9373-f736b9d27886.png)

**Additional context**
I found tf.nn.conv2d_transpose has been tested in [tensorflow-onnx](https://github.com/onnx/tensorflow-onnx) with padding="VALID",while I use padding="SAME".
https://github.com/onnx/tensorflow-onnx/blob/f64772ce166ea2a0402524d9741b2fb71e5663df/tests/test_backend.py#L579-L590

**Comment by hariharans29** [comment]
The PR you are referring to: https://github.com/microsoft/onnxruntime/pull/4627 supports asymmetric padding in CUDA `Conv` kernel (not `ConvTranspose`). We haven't seen too many models requiring asymmetric padding support for `ConvTranspose` in the CUDA EP and it is a backlog item. If more community members require this, it can be prioritized.

**Comment by uvbkq** [comment]
Are there any updates on this item or whether its backlog position has been changed? I believe this might be blocking for a lot of Autocoders, UNets or others requiring the model output size to match its input size when training with Tensorflow?

**Comment by OvervCW** [comment]
For us this is currently making the ONNX runtime unusable because we have a lot of segmentation models that are all taking 12x as long as with TensorFlow/TensorRT.

**Comment by dBeker** [comment]
Is there any progress in this?

**Comment by jensk1** [comment]
Any news on this?

**Comment by disastrousvertical** [comment]
Any updates? I think enough community members have reacted to this already.

**Comment by Toeplitz** [comment]
@hariharans29 do you have any updates on this?

This is blocking us from using some of our existing models in onnx.

2024-02-23 10:12:27.065850685 [W:onnxruntime:, cuda_execution_provider.cc:2319 ConvTransposeNeedFallbackToCPU] Dropping the ConvTranspose node: model/conv3d_transpose/conv3d_transpose to CPU because it requires asymmetric padding which the CUDA EP currently does not support

**Comment by lschaupp** [comment]
+1 bump

**Comment by Positliver** [comment]
+1  same error

**Comment by zainali-AT** [comment]
+1

**Comment by filipix** [comment]
+1 same error

**Comment by andorxornot** [comment]
+1 still exist

**Comment by dayuanzong** [comment]
根因是 ConvTranspose 的 asymmetric padding （pads=[0,0,1,1]），CUDA EP 不支持，导致内部强制回退,在导出脚本里 自动重写模型 ：把这些 ConvTranspose 的 pads 改为对称，并在后面加 Slice 裁掉最后一行/列，保持输出一致，从而彻底避免 CUDA 回退.

[export_xseg_onnx.py](https://github.com/user-attachments/files/25561743/export_xseg_onnx.py)

**Comment by jvallesf** [comment]
+1 
Also interested, the problem still exists in ONNX Runtime 1.24.4 causing inference to be slower on GPU.
In my case I'm converting models from TF to ONNX.
